Tests/ApiEndpoints/test_gordon360_pytest.py

```diff
@@ -50,10 +50,15 @@
 REQUEST_STATUS_DENIED = 'Denied'
 REQUEST_STATUS_PENDING = 'Pending'
 
-# Configuration Details
-username = credentials.username
-password = credentials.password
-my_id_number = credentials.id_number
+
+# # # # # # # # #
+#  Credentials  #
+# # # # # # # # #
+
+username = '360.studenttest'
```

Comment:
You can't put credentials inside github!!!
Please use credentials.username and credentials.password